app.tf

```diff
@@ -36,21 +36,11 @@ resource "volterra_origin_pool" "this" {
   }
 }
 
-resource "volterra_waf" "this" {
-  name        = format("%s-waf", var.web_app_name)
-  description = format("WAF in block mode for %s", var.web_app_name)
-  namespace   = local.namespace
-  app_profile {
-    cms       = []
-    language  = []
-    webserver = []
-  }
-  mode = "BLOCK"
-  lifecycle {
-    ignore_changes = [
-      app_profile
-    ]
-  }
+resource "volterra_app_firewall" "this" {
+  name                     = format("%s-waf", var.web_app_name)
+  description              = format("WAF in block mode for %s", var.web_app_name)
+  namespace                = local.namespace
+  allow_all_response_codes = true
```

Comment:
can you pls check with WAAP team to review this defualt app_firewall config is ok

Reply:
this looks OK to me. It creates a default app firewall in blocking mode.